enclave-runner/src/usercalls/mod.rs

```diff
@@ -267,17 +277,13 @@ enum EnclaveKind {
 }
 
 struct CommandSync {
-    threads: Vec<StoppedTcs>,
     primary_panic_reason: Option<EnclaveAbort<EnclavePanic>>,
     other_reasons: Vec<EnclaveAbort<EnclavePanic>>,
-    running_secondary_threads: usize,
+    threads_queue: crossbeam::queue::SegQueue<StoppedTcs>
```

Comment:
You have a crossbeam queue inside of a `Mutex`, that seems unnecessary. But also, can you unify the way the available TCSes are kept in the `EnclaveState` between libraries and commands?